### PR TITLE
Remove Cloudflare warning

### PR DESCRIPTION
**Pull request by pento**
## Description

We haven't seen reports of Cloudflare blocking REST API requests for a while, they've presumably done some tweaking to how their WAF behaves.

This PR removes the Cloudflare warning added in #8640.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows the accessibility standards.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/api-fetch/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.2.2 (Unreleased)
+
+### Double You Tee Eff, Mate?
```
Comment (tofumatt) [inline packages/api-fetch/CHANGELOG.md]: Um, wut?

**Review by tofumatt — CHANGES_REQUESTED** [review]
I’m not sure anyone will know what those changelog messages are, so it’s best to remove them I think.

Side-note: I thought Gutenberg meant to move all its PHP to core? Is this file sticking around?

Otherwise looks good.

File: packages/api-fetch/CHANGELOG.md (hunk 1 above)
Comment (pento) [reply packages/api-fetch/CHANGELOG.md]: [](https://www.youtube.com/watch?v=kCpjgl2baLs)

**Comment by pento** [comment]
Okay, changelog entries are gone. 🙂

PHP files have all been merged to core, but this warning wasn't: it needed to be removed from the JS, so I've removed it from the PHP for completeness.

File: packages/api-fetch/CHANGELOG.md (hunk 1 above)
Comment (tofumatt) [reply packages/api-fetch/CHANGELOG.md]: I mean, I'm not *that* young! But just didn't know what it had to do with a changelog 😅

**Review by tofumatt — APPROVED** [review]
Thanks!

more-red-than-green

File: packages/api-fetch/CHANGELOG.md (hunk 1 above)
Comment (pento) [reply packages/api-fetch/CHANGELOG.md]: At this stage in the project, classic memes are the only way to communicate. 😉

**Pull request merged**